src/sidebar/helpers/version-data.js

```diff
@@ -37,7 +37,7 @@ export default class VersionData {
       }
     }
 
-    this.version = '__VERSION__'; // replaced by versionify
+    this.version = '__VERSION__';
```

Comment:
So what _does_ happen now?

Reply:
@robertknight added a Rollup plugin that allows to replace certain strings for specific values. See `rollup.config.mjs`